docs/source/tutorials/get-metadata.rst

```diff
@@ -0,0 +1,190 @@
+we intend to do (i.e. which scan type or which data processing routine), who is
+doing it, and any additional context like sample information.
+
+The only fields *guaranteed* by Databroker to be present are ``uid`` (a
+globally unique identifer for the Run) and ``time`` (when it started) but there
+is often a great deal more.
+
+.. code:: python
+
+   >>> run.metadata["start"]
+   Start({
+   'XDI': {'Beamline': {'collimation': 'paraboloid mirror, 5 nm Rh on 30 nm Pt',
+                      'focusing': 'not in use',
+                      'harmonic_rejection': 'flat mirror, Pt stripe, pitch = '
+                                            '7.0 mrad relative to beam',
+                      'name': 'BMM (06BM) -- Beamline for Materials '
+                              'Measurement',
+                      'xray_source': 'NSLS-II three-pole wiggler'},
+         'Column': {},
+         'Detector': {'I0': '10 cm N2', 'Ir': '25 cm N2', 'It': '25 cm N2'},
+         'Element': {'edge': 'K', 'symbol': 'Ni'},
+         'Facility': {'GUP': 305832,
+                      'SAF': 305669,
+                      'current': '399.6',
+                      'cycle': '2020-1',
+                      'energy': '3.0',
+                      'mode': 'top-off',
+                      'name': 'NSLS-II'},
+         'Mono': {'angle_offset': 16.058109,
+                  'd_spacing': '3.1353241',
+                  'direction': 'forward',
+                  'encoder_resolution': 5e-06,
+                  'name': 'Si(111)',
+                  'scan_mode': 'fixed exit',
+                  'scan_type': 'step'},
+         'Sample': {'name': 'Ni', 'prep': 'Ni foil in ref'},
+         'Scan': {'edge_energy': 8332.800000000001,
+                  'experimenters': 'Neil Hyatt, Martin Stennett, Dan Austin, '
+                                   'Seb Lawson'},
+
+    ...  # snipped for brevity
+    }
+
+How It Ended
+------------
+
+There are other things we can only know at the **stop** (end) of an experiment,
+including when and how it finished and how many events (rows) of data were
+collected in each stream.
+
+.. ipython:: python
+
+   run.metadata["stop"]
+
+We can use this to print the unique IDs of any experiments that failed
+
+.. ipython:: python
+
+   for uid, run in catalog.items():
+       if run.metadata["stop"]["exit_status"] != "success":
+           print(f"Run {uid} failed!")
+
+or, getting a bit fancier, to tally the number of failures.
+
+.. ipython:: python
+
+   from collections import Counter
+
+   counter = Counter()
+   for _, run in catalog.items():
+       counter.update({run.metadata["stop"]["exit_status"]: 1})
+   counter
+
+TO DO: Obtain an example catalog that has some failures in it so that this
+example is not so trivial.
+
+Low-level Hardware Timestamps
+-----------------------------
+
+.. note::
+
+   We mention this for completeness, but you will probably never need it.
```

Comment:
This seems too strong, I think using these for (~.1s precision) alignment should be fair-game.